Dockerfile.ubi.cpu.ppc64le

```diff
@@ -0,0 +1,247 @@
+        -DARROW_JEMALLOC=ON \
+        -DARROW_BUILD_STATIC="OFF" \
+        -DARROW_PARQUET=ON \
+        .. && \
+    make install -j ${MAX_JOBS:-$(nproc)} && \
+    cd ../../python/ && \
+    uv pip install -v -r requirements-wheel-build.txt && \
+    PYARROW_PARALLEL=${PYARROW_PARALLEL:-$(nproc)} \
+    python setup.py build_ext \
+    --build-type=release --bundle-arrow-cpp \
+    bdist_wheel --dist-dir /arrowwheels/
+
+###############################################################
+# Stage to build opencv
+###############################################################
+FROM base-builder AS cv-builder
+
+ARG MAX_JOBS
+ARG OPENCV_VERSION=84
+ARG ENABLE_HEADLESS=1
+RUN --mount=type=cache,target=/root/.cache/pip \
+    --mount=type=cache,target=/root/.cache/uv \
+    source /opt/rh/gcc-toolset-13/enable && \
+    git clone --recursive https://github.com/opencv/opencv-python.git -b ${OPENCV_VERSION} && \
+    cd opencv-python && \
+    sed -i 's/"setuptools==59.2.0",/"setuptools<70.0",/g' pyproject.toml && \
+    python -m build -w -o /opencvwheels/
+
+###############################################################
+# Stage to build & install vllm
+###############################################################
+# Why is this stage needed?
+# There are packages which vllm depends on which build from src provided compilers are present
+# To keep the final image as small as possible, we can have all the wheels build in this stage
+# and consume it in the final stage
+FROM base-builder AS vllm-deps
+
+ARG VLLM_TGIS_ADAPTER_VERSION=0.6.2
+ARG TGIS_PATCH=178d4baf
+ARG GRPC_PYTHON_BUILD_SYSTEM_OPENSSL=1
+
+# create artificial dependencies between stages
+# wait for torch, arrow & openv build stages
+COPY --from=torch-builder /tmp/control /dev/null
+COPY --from=arrow-builder /tmp/control /dev/null
+COPY --from=cv-builder /tmp/control /dev/null
+
+# build vllm wheel
+# Note: use pip instead of uv to build the pip cache
+ARG VLLM_TARGET_DEVICE=cpu
+RUN --mount=type=cache,target=/root/.cache/pip \
+    --mount=type=cache,target=/root/.cache/uv \
+    --mount=type=bind,from=torch-builder,source=/torchwheels/,target=/torchwheels/,ro \
+    --mount=type=bind,from=arrow-builder,source=/arrowwheels/,target=/arrowwheels/,ro \
+    --mount=type=bind,from=cv-builder,source=/opencvwheels/,target=/opencvwheels/,ro \
+    --mount=type=bind,src=.,dst=/vllm,rw \
+    source /opt/rh/gcc-toolset-13/enable && \
+    pip install -v \
+        /arrowwheels/*.whl \
+        /opencvwheels/*.whl \
+        /torchwheels/*.whl && \
+    cd /vllm && \
+    pip install pandas==2.0 && \
+    pip install -v ".[tensorizer]" --no-build-isolation && \
+    python -m build -w -o /wheelsdir/ --no-isolation && \
+    cd .. && \
+    git clone https://github.com/opendatahub-io/vllm-tgis-adapter.git -b ${VLLM_TGIS_ADAPTER_VERSION} && \
+    cd vllm-tgis-adapter && \
+    git cherry-pick --no-commit ${TGIS_PATCH} && \
+    python -m build -w -o /wheelsdir/
```

Comment:
Thinking about this some more, I don't think we need to build the adapter from scratch, `uv pip install vllm-tgis-adapter==${VLLM_TGIS_ADAPTER_VERSION}` should work. 

As a sidenote, since `vllm-tgis-adapter` depends on `vllm`, it's better to install all python dependencies in a single `install command`, to avoid the possibility overwriting installed versions and getting an error if something is not compatible.